src/app/pages/global-reach/map/mapdropdown.js

```diff
@@ -112,19 +113,40 @@ export default class Mapdropdown extends Controller {
         document.getElementById('back-search-div').setAttribute('style', 'display: none;');
         document.getElementById('back-result-div').setAttribute('style', 'display: block;');
     }
+    generateCityState(city, state) {
```

Comment:
This is duplicated from `map.js`. It really doesn't need to be a method at all; it's just a helper function. It can be defined as a separate exportable in `map.js` and then imported like
```
import {generateCityState} from './map.js';
```
here.